tests/jobserver/views/test_admin.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.urls import reverse
+
+from jobserver.models import Backend, User
+from jobserver.views.admin import ApproveUsers
+
+from ...factories import OrgFactory, UserFactory
+
+
+@pytest.mark.django_db
+def test_approveusers_as_non_superuser(rf):
+    request = rf.get("/")
+    request.user = UserFactory()
+
+    response = ApproveUsers.as_view()(request)
+
+    assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_approveusers_get_success(rf, superuser):
+    request = rf.get(f"/?user={superuser.pk}")
+    request.user = superuser
+
+    response = ApproveUsers.as_view()(request)
+
+    assert response.status_code == 200
+
+    expected = set(User.objects.filter(pk=superuser.pk))
+    output = set(response.context_data["users"])
+    assert output == expected
```

Comment:
Could you do `assert list(response.context_data["users"]) == [superuser]`?

Reply:
Oh! I hadn't thought about wrapping the output in a list - YES